### PR TITLE
Data Lake vs. Data Warehouse vs. Database

### LINKED ISSUE #682: Data Lake vs Data Warehouse vs Database
### Brief Summary: 
Describe the what, why, and how of your content idea in 2-5 sentences.

Every industry needs to process data. But the kind of data, its scope, and its use will
illustrate if a data warehouse, database, or a data lake will be best solution for your enterprise. This article will compare these three data management systems. 
### Key Takeaways: 
What are the 3-5 most important things the reader should understand or be able to do after reading this article?
1. An overview of data lake, data warehouse, and database. 
2. Industrial use cases of data lake, data warehouse, and database. 
3. The future of data lakes, data warehouses, and databases. 
### References: 
Please list links to any published content/research that you intend to use to support/guide this article. (If none, please indicate 'N/A'.)
N/A

### PR DESCRIPTION
**Pull request by EKahuha**
this fixes #682

#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.

**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process and style-once published, you can then submit any number of articles.)**
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

### CODE REVIEW

**Comment by hectorkambow** [comment]
Plagiarism check done

File: articles/data-lake-vs-data-warehouse-vs-database/index.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+Most of today&#39;s organizations collect and process large amounts of data daily. But this data can only bring value if the organization derives valuable insights from the data. One of the dependable factors for big data strategies&#39; success is choosing the right technology to enable data storage, search, analysis, and report generation.
+
+Depending on the kind of data, its use, and scope, an organization can use a data warehouse, data lake, or database. This article will discuss data lake, data warehouse, database, their use cases, and future outlook to help you decide which best suits your goals, data strategy, and infrastructure.
+
+### **An overview of data lake**
```
Comment (lalith1403) [inline articles/data-lake-vs-data-warehouse-vs-database/index.md:5]: No need to bolden the headings.

Comment (lalith1403) [reply articles/data-lake-vs-data-warehouse-vs-database/index.md:5]: Make changes throughout the article, wherever applicable.

File: articles/data-lake-vs-data-warehouse-vs-database/index.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+Besides, DBMS ensures the security and protection of databases and maintains data consistency in multiple users.
+
+Some of the most popular DBMS are [MongoDB](https://www.mongodb.com/), [Oracle](https://www.oracle.com/database/), [Apache Cassandra](http://cassandra.apache.org/), [PostgreSQL](https://www.postgresql.org/), [Elasticsearch](https://www.elastic.co/), and [Redis](https://redis.io/).
+
+### **Industrial use cases of data warehouses**
+
+Industries that process large amounts of data find data warehousing most applicable to their needs. They include governments and companies in the insurance, healthcare, education, and finance industries.
+
+- **Healthcare**
```
Comment (lalith1403) [inline articles/data-lake-vs-data-warehouse-vs-database/index.md:39]: Make this an H4 heading. Change accordingly throughout the article.

File: articles/data-lake-vs-data-warehouse-vs-database/index.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+Telecommunication companies use databases to store and generate customer bills, balances for prepaid customers, call logs, among other essential information.
+
+- **Sales**
+
+The sales department benefits significantly from a company&#39;s database. Among other tasks, sales teams use databases to track sales, product performance, and customer information.
+
+### **The future of data lakes, data warehouses, and databases**
+
+The current shift towards cloud-based data platforms to mitigate data issues and manage data suggests that data lakes&#39; future is the cloud. The cloud is scalable and expandable, thus expected to attract more users.
```
Comment (lalith1403) [inline articles/data-lake-vs-data-warehouse-vs-database/index.md:77]: When you say the future of data lakes is in the cloud, isn't it already the case? I feel you should elaborate on how data lakes are being deployed via the cloud. That would make more sense.

**Review by lalith1403 — CHANGES_REQUESTED** [review]
A few minor changes and this should be good to go. Congratulations on a well written article.

**Comment by EKahuha** [comment]
@lalith1403 
Done.

**Review by lalith1403 — APPROVED** [review]

**Pull request merged**